.gitignore

```diff
@@ -128,3 +128,6 @@ dist
 .yarn/build-state.yml
 .yarn/install-state.gz
 .pnp.*
+
+# package lock
+package-lock.json
```

Comment:
Don't add `package-lock.json` to `.gitignore`

Reply:
Fixed.